src/action/handle.rs

```diff
@@ -88,6 +88,9 @@ pub fn switch(ctx: &mut Ctx, mut args: SwitchArgs) {
         if handle == "-" {
             if let Ok(previous_handle) = StateField::PreviousEndpoint.get(ctx) {
                 handle = previous_handle;
+            } else {
+                eprintln!("No previous handle was found");
```

Comment:
Errors should be all lowercase. Also, I think it makes more sense to panic here instead of returning. Otherwise, we won't have the correct status code on output. I'd also shorten the message to "no previous handle" instead.